src/mon/MonCommands.h

```diff
@@ -276,6 +276,7 @@ COMMAND("mon versions",
 COMMAND("versions",
 	"check running versions of ceph daemons",
 	"mon", "r", "cli,rest")
+COMMAND("mon status", "print status of monitors", "mon", "r", "cli,rest")
```

Comment:
as you noted in the commit message, we have already `mon_status`, which offers basically the same set of information provided by `mon_status`, aside from that `mon_status` prints the "name" not id of monitors.

i am afraid that duplicating them with this subtle difference with existing command would confuse our user.

Reply:
the main difference between `mon_status` and this command is the user interface just as the commit message, or I misunderstand the meaning of  [tracker](http://tracker.ceph.com/issues/24232) and [trello](https://trello.com/c/PTgwBpmx/363-mon-ceph-svc-status-commands)?

Reply:
@hsiaoyinn @tchaikov the tracker's scope is, unfortunately, way more limited than the scope of the trello card. And while the tracker ticket is not assigned to me, the trello card is and is on my queue (planned to start working on it next week).

The idea is to present service-specific information on their `status` commands. E.g., `ceph mon status`  would present information about the `mon` service (how many we have, where they are, if they are in quorum, required features, health, possible issues with the monitors), whereas a `mds status` or an `rgw status`  would provide information about those services instead.

As it stands, this PR could still be a move forward towards achieving the trello card's purpose. However, what is being done here is a plain-text representation of `mon_status`, and it introduces additional technical debt by partially duplicating the work of `mon_status`.

For this PR to move forward, I would suggest two things, should you want to proceed:

1. work on `Monitor::get_mon_status()` to allow `mon_status`  to understand a `--format plain`, although defaulting to `json` if the formatter is not specified. We must ensure that the behavior of `mon_status` by default is the same as we have today.

2. call on `get_mon_status()` when calling `mon status`, allow it to understand a `--format`.

If the PR achieves these constraints (and assuming `mon_status` is still compatible with the current behavior), I'm okay with merging the code. Be warned that this will likely suffer further changes once we move towards the scope of the trello card though.

Reply:
@jecluis it's impossible to let `get_mon_status()`  understand a `--format plain` without additional parameter, can I add more parameter in `get_mon_status()`? I'll ensure `mon_status` is still compatible with the current behavior.